tsl/test/expected/compression_bgw.out

```diff
@@ -223,3 +223,158 @@ SELECT COUNT(*) AS dropped_chunks_count
+ Fri Jan 03 04:00:00 2020 PST | 11 |     2
+(3 rows)
+
+ 
+--check status for chunk --
+SELECT chunk_status,
+       chunk_name as "CHUNK_NAME"
+FROM compressed_chunk_info_view 
+WHERE hypertable_name = 'test2' ORDER BY chunk_name;
+ chunk_status |     CHUNK_NAME     
+--------------+--------------------
+            3 | _hyper_10_48_chunk
+(1 row)
+
+SELECT compressed_chunk_schema || '.' || compressed_chunk_name as "COMP_CHUNK_NAME",
+        chunk_schema || '.' || chunk_name as "CHUNK_NAME"
+FROM compressed_chunk_info_view 
+WHERE hypertable_name = 'test2' \gset
+SELECT count(*) from test2;
+ count 
+-------
+    29
+(1 row)
+
+SELECT recompress_chunk(:'CHUNK_NAME'::regclass); 
+             recompress_chunk             
+------------------------------------------
+ _timescaledb_internal._hyper_10_48_chunk
+(1 row)
+
+SELECT chunk_status,
+       chunk_name as "CHUNK_NAME"
+FROM compressed_chunk_info_view 
+WHERE hypertable_name = 'test2' ORDER BY chunk_name;
+ chunk_status |     CHUNK_NAME     
+--------------+--------------------
+            1 | _hyper_10_48_chunk
+(1 row)
+
+--- insert into a compressed chunk again + a new chunk--
+INSERT INTO test2 values ( '2020-01-03 11:01:03+00', 20, 11, '33row'), 
+                         ( '2020-01-03 11:01:06+00', 20, 11, '36row'), 
+                         ( '2020-01-03 11:02:00+00', 20, 12, '12row'), 
+                         ( '2020-04-03 00:02:00+00', 30, 13, '3013row'); 
+SELECT time_bucket(INTERVAL '2 hour', timec), b, count(*)
+FROM test2
+GROUP BY time_bucket(INTERVAL '2 hour', timec), b
+ORDER BY 1, 2;
+         time_bucket          | b  | count 
+------------------------------+----+-------
+ Fri Jan 03 02:00:00 2020 PST | 11 |    28
+ Fri Jan 03 02:00:00 2020 PST | 12 |     2
+ Fri Jan 03 04:00:00 2020 PST | 11 |     2
+ Thu Apr 02 17:00:00 2020 PDT | 13 |     1
+(4 rows)
+
+--chunk status should be unordered for the previously compressed chunk
+SELECT chunk_status,
+       chunk_name as "CHUNK_NAME"
+FROM compressed_chunk_info_view 
+WHERE hypertable_name = 'test2' ORDER BY chunk_name;
+ chunk_status |     CHUNK_NAME     
+--------------+--------------------
+            3 | _hyper_10_48_chunk
+            0 | _hyper_10_51_chunk
+(2 rows)
+
+SELECT add_compression_policy AS job_id
+  FROM add_compression_policy('test2', '30d'::interval) \gset
+CALL run_job(:job_id);
+CALL run_job(:job_id);
+-- status should be compressed ---
+SELECT chunk_status,
+       chunk_name as "CHUNK_NAME"
+FROM compressed_chunk_info_view 
+WHERE hypertable_name = 'test2' ORDER BY chunk_name;
+ chunk_status |     CHUNK_NAME     
+--------------+--------------------
+            1 | _hyper_10_48_chunk
+            1 | _hyper_10_51_chunk
+(2 rows)
+
+\set ON_ERROR_STOP 0
+-- call recompress_chunk when status is not unordered
+SELECT recompress_chunk(:'CHUNK_NAME'::regclass, true); 
+psql:include/recompress_basic.sql:95: NOTICE:  chunk "_hyper_10_48_chunk" is already compressed
```

Comment:
I find this error confusing. When recompressing, you'd expect the chunk to be compressed already, so why is it giving an error on this?


Maybe it should say something like this:

```suggestion
psql:include/recompress_basic.sql:95: NOTICE:  nothing to recompress in chunk "_hyper_10_48_chunk"
```

Reply:
fixed.